app/models/manageiq/providers/vmware/cloud_manager/refresh_parser.rb

```diff
@@ -134,13 +134,30 @@ def parse_vm(vm)
       }
     end
 
+    snapshot_resp = vm.service.get_snapshot_section(vm.id).data[:body][:Snapshot]
+
+    unless snapshot_resp.nil?
+      snapshot = [
+        {
+          :name        => "#{vm.name}(snapshot)",
+          :uid         => "#{vm.id}-#{snapshot_resp[:created]}",
```

Comment:
Yes I have checked, `href` only contains id of VM.

Reply:
Maybe use a different seperator than `'-'` since that is part of the uuid already?  That way you can split the uid apart with something like `snapshot.uid.split("__")`